docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+services:
+  api:
+    tty: true
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    volumes:
+      - ./src:/opt/app/src
+    ports:
+      - 3001:3001
+    environment:
+      - SHELL=/bin/sh
```

Comment:
How will we pass other env variables to the app?

Reply:
'.env'

Reply:
That ok?